plugins/woocommerce/src/Internal/EmailEditor/patterns/WooMailContentPattern.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types = 1);
+
+namespace Automattic\WooCommerce\Internal\EmailEditor\Patterns;
+
+use MailPoet\EmailEditor\Engine\Patterns\Abstract_Pattern;
+
+class WooMailContentPattern extends Abstract_Pattern {
+	public $name = 'woo-mail-content-pattern';
```

Comment:
Sry for nitpicking. I would use WooEmail... and woo-email to be consistent.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/55754/commits/c9a010081c8f3a28d53f4bbe61f996abcbfd784b